lib/jsonlint/cli.rb

```diff
@@ -11,7 +11,7 @@ def execute!
 
       files_to_check = @argv
 
-      Trollop::die 'nead at least one JSON file to check' if files_to_check.empty?
+      Trollop::die 'need at least one JSON file to check' if files_to_check.empty?
```

Comment:
This probably should be changed from Trollop::die to Trollop.die while you are at it :)